relay-server/src/endpoints/envelope.rs

```diff
@@ -48,7 +48,7 @@ fn store_envelope(
     request: HttpRequest<ServiceState>,
 ) -> ResponseFuture<HttpResponse, BadStoreRequest> {
     let content_type = request.content_type();
-    if content_type != ContentType::Envelope {
+    if !content_type.is_empty() && content_type != ContentType::Envelope {
```

Comment:
Unfortunately setting the Content-Type to the empty string is not a valid solution to avoid CORS preflight requests 😢 

In the screenshot, the request headers were logged to the console.

![image](https://user-images.githubusercontent.com/88819/80603664-03342100-8a31-11ea-9f2a-680de533bede.png)

Reply:
The requirement on Content-Type seems to be as strict as listed in [CORS Simple Requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Simple_requests):


> The only allowed values for the Content-Type header are:
> - application/x-www-form-urlencoded
> - multipart/form-data
> - text/plain

Reply:
:'(